meta-resin-common/recipes-connectivity/resin-proxy-config/resin-proxy-config/redsocks.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
```

Comment:
Given that the service depends on the daemon configuration, I agree that we can leave the service as being resin specific (eg if the configuration uses daemon on/off needs to be reflected in the service accordingly).